datalad/tests/test_tests_utils.py

```diff
@@ -155,27 +151,6 @@ def t():  # pragma: no cover
         t()
 
 
-def test_with_testrepos():
```

Comment:
I would have kept this test for the duration of having `@with_testrepos` in the codebase.

Reply:
I agree.

Reply:
Done.